lcd_test/helpers_test.go

```diff
@@ -307,14 +306,15 @@ func doTransfer(
 // signed and broadcasted. The sending account's number and sequence are
 // determined prior to generating the tx.
 func doTransferWithGas(
-	t *testing.T, port, seed, name, memo, pwd string, addr sdk.AccAddress,
+	t *testing.T, port, seed, name, memo string, addr sdk.AccAddress,
 	gas string, gasAdjustment float64, simulate, broadcast bool, fees sdk.Coins,
+	kb crkeys.Keybase,
 ) (resp *http.Response, body string, receiveAddr sdk.AccAddress) {
 
 	// create receive address
-	kb := crkeys.NewInMemory()
+	kb2 := crkeys.NewInMemory()
```

Comment:
do we need to create a new instance here?

Reply:
This may be a change introduced inadvertently by my IDE. Will investigate shortly and eventually respond

Reply:
Ok no, I've just updated the reference name cause `kb` is now variable's name taken as a function's parameter.
The temporary keybase seems to be created to generate the body of the request. In all honesty I can't tell more, I've just updated the logic to work with the new keyring